### ISSUE TITLE
Login/Logout icons

### ISSUE THREAD

**Issue by pbhowmic** [Templarian/MaterialDesign#288]
For the login and logout icons, the door width is inconsistent.

**Comment by Templarian** [comment]
I'll look into this when I get a moment.

Thanks for opening an issue.

**Comment by pbhowmic** [comment]
Actually, you can close this one. It is pretty minor, I can only see a slight deviation at an 8x resolution in GIMP. Check it all the same

**Comment by Templarian** [comment]
I looked at it earlier and I'm not seeing any variation. You are using the SVG files from MaterialDesignIcons.com right and not out of the repo?

**Comment by pbhowmic** [comment]
Right. And the difference is (from left side boundary to right side boundary) about 1 pixel. login is almost 14 pixels wide, logout 13.

**Comment by Templarian** [comment]
Oh, yea, that's just to keep door the same size and the spacing even between the two. I illustrated how I determined the spacing below...

![spacing](https://cloud.githubusercontent.com/assets/338885/7196981/a4e59efc-e4a0-11e4-8994-33748daaa2e2.png)

Legend: Gray = recommended padding (2dp), Red = eye balling spacing, other = spacing comparisons.

I'm sure there are other ways this icon could be designed though to more evenly handle the visual alginement. Maybe in the future it could be replaced with less of a literal door in/out concept. :smile:

**Comment by CoDEmanX** [comment]
The spacing / door width is equal, but it doesn't look that way. To improve visual equality, I suggest to move the right vertical stroke of the login door by 1 pixel to the right.

Currently:
![login_stock](https://cloud.githubusercontent.com/assets/1309249/12393042/797b9302-bdf3-11e5-860c-a25f496928ce.png) ![logout_stock](https://cloud.githubusercontent.com/assets/1309249/12393041/797b7e6c-bdf3-11e5-84fa-15210c364296.png)

Improved:
![login](https://cloud.githubusercontent.com/assets/1309249/12393072/ab966042-bdf3-11e5-8a38-992850dcf96e.png) ![logout_stock](https://cloud.githubusercontent.com/assets/1309249/12393041/797b7e6c-bdf3-11e5-84fa-15210c364296.png)

It shouldn't be much of the problem to do that, because the horizontal placement of both icons is different and the varying door width thus not apparent even if you use them for a hover effect.

**Comment by JamesCoyle** [comment]
Could also keep the arrow on the same side maybe? 
![image](https://cloud.githubusercontent.com/assets/10384436/26384033/60d443bc-402e-11e7-85ae-14da4761b0d1.png)![image](https://cloud.githubusercontent.com/assets/10384436/26384038/6f6bdcd2-402e-11e7-853d-5d26c2d0dd40.png)

Note those are just quick pixel based drawings not smooth SVG.

**Comment by goyney** [comment]
To add to this, Font Awesome's icons for sign in and out follow our current pattern, but both icons are exactly 16dp in width. So, my goal when opening our two versions was to find a solution that takes the exact same width and is symmetrically consistent.

First, I found some grid issues in both around the arrow head, so those were resolved. I then opened the gap in the door 2dp in both directions and lined the arrow up adjacent to the door border. These are what I ended up with. (The grey outline is the current icons for comparison.)

![image](https://user-images.githubusercontent.com/1015040/48816596-bf53cd00-ed10-11e8-8f83-0d12308dd203.png)![image](https://user-images.githubusercontent.com/1015040/48816604-d09cd980-ed10-11e8-81b7-cef2d1d99bf2.png)

I feel these are more consistent and would resolve the unbalanced feeling you currently get when looking at the two.

[loginout.zip](https://github.com/Templarian/MaterialDesign/files/2602493/loginout.zip)
